packages/lighthouse-service/.eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
   },
   plugins: ["@typescript-eslint"],
   rules: {
-    "max-len": ["error", { code: 130 }],
+    "max-len": [ "error", { code: 130 } ],
+    "import/prefer-default-export": "off"
```

Comment:
```suggestion
```
https://airbnb.io/javascript/#:~:text=back%20to%20top-,Modules,-10.1%20Always%20use

Reply:
https://blog.piotrnalepa.pl/2020/06/26/default-exports-vs-named-exports/

A lot more articles 😅 . It's a nightmare